src/atomate2/aims/schemas/calculation.py

```diff
@@ -0,0 +1,492 @@
+    cbm: float
+        The conduction band minimum in eV (if system is not metallic
+    vbm: float
+        The valence band maximum in eV (if system is not metallic)
+    atomic_steps: list[Structure or Molecule]
+        Structures for each ionic step"
+    """
+
+    energy: float = Field(
+        None, description="The final total DFT energy for the calculation"
+    )
+    energy_per_atom: float = Field(
+        None, description="The final DFT energy per atom for the calculation"
+    )
+
+    structure: Union[Structure, Molecule] = Field(
+        None, description="The final structure from the calculation"
+    )
+
+    efermi: float = Field(
+        None, description="The Fermi level from the calculation in eV"
+    )
+
+    forces: Optional[list[Vector3D]] = Field(
+        None, description="Forces acting on each atom"
+    )
+    all_forces: Optional[list[list[Vector3D]]] = Field(
+        None,
+        description="Forces acting on each atom for each structure in the output file",
+    )
+    stress: Optional[Matrix3D] = Field(None, description="The stress on the cell")
+    stresses: Optional[list[Matrix3D]] = Field(
+        None, description="The atomic virial stresses"
+    )
+
+    is_metal: Optional[bool] = Field(None, description="Whether the system is metallic")
+    bandgap: Optional[float] = Field(
+        None, description="The band gap from the calculation in eV"
+    )
+    cbm: float = Field(
+        None,
+        description="The conduction band minimum, or LUMO for molecules, in eV "
+        "(if system is not metallic)",
+    )
+    vbm: Optional[float] = Field(
+        None,
+        description="The valence band maximum, or HOMO for molecules, in eV "
+        "(if system is not metallic)",
+    )
+    atomic_steps: list[Union[Structure, Molecule]] = Field(
+        None, description="Structures for each ionic step"
+    )
+
+    @classmethod
+    def from_aims_output(
+        cls,
+        output: AimsOutput,  # Must use auto_load kwarg when passed
+        store_trajectory: bool = False,
+    ) -> CalculationOutput:
+        """
+        Create an FHI-aims output document from FHI-aims outputs.
+
+        Parameters
+        ----------
+        output: .AimsOutput
+            An AimsOutput object.
+        store_trajectory: bool
+            A flag setting to store output trajectory
+
+        Returns
+        -------
+        The FHI-aims calculation output document.
+        """
+        structure = output.final_structure
+
+        electronic_output = {
+            "efermi": output.fermi_energy,
+            "vbm": output.vbm,
+            "cbm": output.cbm,
+            "bandgap": output.band_gap,
+            "direct_bandgap": output.direct_band_gap,
+        }
+
+        forces = None
+        if output.forces is not None:
+            forces = output.forces
+
+        stress = None
+        if output.stress is not None:
+            stress = voigt_to_full_stress_conv(output.stress).tolist()
```

Comment:
`output.stress` is `Matrix3D`, but the input to `voigt_to_full_stress_conv` is `Sequence[float]`

Reply:
This is fixed